src/avram/attribute.cr

```diff
@@ -1,4 +1,5 @@
 class Avram::Attribute(T)
+  alias ErrorMessage = String | Proc(String, String, String) | Avram::CallableErrorMessage
```

Comment:
Extract a type alias  that we can use in `Avram::Validations` to give better error message and make it clearer what you can use for a custom message in validations